test/provisioner/provision_azure.go

```diff
@@ -37,15 +37,13 @@ func createResourceGroup() error {
 	if AzureProps.IsCIManaged {
 		log.Infof("Resource group %q is CI managed. No need to create new one manually.", AzureProps.ResourceGroupName)
 
-		resp, err := AzureProps.ResourceGroupClient.Get(context.Background(), AzureProps.ResourceGroupName, nil)
+		_, err := AzureProps.ResourceGroupClient.Get(context.Background(), AzureProps.ResourceGroupName, nil)
 		if err != nil {
 			err = fmt.Errorf("getting resource group %s: %w", AzureProps.ResourceGroupName, err)
 			log.Errorf("%v", err)
 			return err
 		}
 
-		AzureProps.ResourceGroup = &resp.ResourceGroup
```

Comment:
We need this, this was added in #1292 because the resource group object is later used in `azure_test.go` to do some assertions and this could panic due to nil pointer exception, since the object would be empty: https://github.com/confidential-containers/cloud-api-adaptor/blob/27af578f30e6d4f5151f6ee807e33e794e2edfde/test/e2e/azure_test.go#L73-L75

Reply:
Oh I take it back, probably it was solved using the property before.